e2e/paths/uploadDocuments_test.js

```diff
@@ -7,19 +7,25 @@ Before((I, caseViewPage) => {
   caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
 });
 
-Scenario('Selecting to follow for a document produces a textarea', (I, uploadDocumentsPage) => {
-  I.selectOption(uploadDocumentsPage.fields.socialWorkChronologyStatus, 'To follow');
-  I.fillField(uploadDocumentsPage.fields.socialWorkChronologyReason, 'mock reason');
+Scenario('Completing half the Annex documents form by selecting social work chronology document to follow in the c110a application', (I, uploadDocumentsPage, caseViewPage) => {
```

Comment:
I would just take out the first half of the question so it reads something like 'selecting social work chronology document to follow in the c110a application'

Reply:
changed